src/metabase/lib/core.cljc

```diff
@@ -12,6 +12,7 @@
    [metabase.lib.common :as lib.common]
    [metabase.lib.database :as lib.database]
    [metabase.lib.expression :as lib.expression]
+   [metabase.lib.fe-util :as lib.fe-util]
```

Comment:
Introduced this namespace because there are functions very specific for the FE, but at the same time would deserve proper testing. The function added in this PR doesn't fit well into the existing namespaces because it needs functionality from `field`, `join` and `filter` which results in a cyclic reference. Things could be refactored with some effort, but I'm not sure it's worth it.